.gitignore

```diff
@@ -166,3 +166,6 @@ TAGS
 # Do not check in compiled binaries
 semgrep/semgrep/bin/semgrep-core
 semgrep/semgrep/bin/spacegrep
+
+# Git repo that's not a submodule
+/ocaml-tree-sitter
```

Comment:
Not super convinced of having this special case. I d rather have a submodule like ocaml-treesitter-lib-and-rntime

Reply:
hmm, maybe we should have a bare-bones git repo for the ocaml code of ocaml-tree-sitter, and use it as submodule in what's currently ocaml-tree-sitter.

Reply:
Exactly!

Reply:
It requires a bit of restructuring otherwise I'd do it right away.